target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -360,6 +360,9 @@ tplink,tl-wr842n-v2)
 	ucidef_set_led_switch "lan3" "LAN3" "green:lan3" "switch0" "0x10"
 	ucidef_set_led_switch "lan4" "LAN4" "green:lan4" "switch0" "0x02"
 	;;
+tplink,tl-wa1201-v2)
+	ucidef_set_led_netdev "lan" "LAN" "green:lan" "eth0"
+	;;
```

Comment:
I bet this can be merged with some other node.

Reply:
ok